### DISCUSSION TITLE
2024/03/03/Mendelian_randomization_by_TwoSampleMR/

### DISCUSSION THREAD

**Discussion by giscus** [cyclinbox/cyclinbox.github.io#16] [General]
# 2024/03/03/Mendelian_randomization_by_TwoSampleMR/

如题。

https://wz.anoms.top/2024/03/03/Mendelian_randomization_by_TwoSampleMR/

**Comment by jingling000** [comment]
偷图来了，谢谢皖豫小哥

**Reply by cyclinbox** [reply]
不用谢awa
